node-util/sbin/oo-init-quota

```diff
@@ -79,11 +79,11 @@ QUOTA_OPTIONS=usrjquota=aquota.user,jqfmt=vfsv0
 function update_fstab() {
     # ORIGIN_MOUNTPOINT=$1
 
-    if ! tr -s ' ' < /etc/fstab | egrep "^[^#].+ $1 ext4 [^ ]+$QUOTA_OPTIONS "; then
+    if ! tr -s '[:blank:]' < /etc/fstab | egrep "^[^#].+\s+$1\s+ext4\s+.*$QUOTA_OPTIONS\s+"; then
```

Comment:
The `.*` before `$QUOTA_OPTIONS` will pick up any character, even spaces. This should probably be `[^ ]*` or even better `[^\h]` (which is shorthand for `[^[:blank:]]`). The `[^ ]+` previously in this line is odd and probably incorrect. It requires a non-space character before the quota options when it is possible that the quota options may be first in the list of options.